assets/src/edit-story/theme.js

```diff
@@ -105,6 +105,7 @@ const theme = {
       v14: '#3E445B',
       v15: '#212433',
       v16: '#2C2C2C',
+      v17: '#5e615c',
```

Comment:
As this is known as `gray24` in the color scheme, can we call it that here too (like [the other named colors](https://github.com/google/web-stories-wp/pull/3787/files#diff-e9f19c950bf7580cdde6db2b8ffb5ea5R119-R120))?

Also, it's technically a foreground color (as per the design guide), so it should be moved to the `fg` section.

![Screen Shot 2020-08-20 at 19 34 31](https://user-images.githubusercontent.com/637548/90836128-9369df00-e31c-11ea-84b3-cc26b5b6339b.png)